PubnativeLite/PubnativeLite/Banner/VWAdvertView.m

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+#import "VWAdvertView.h"
+
+@implementation VWAdvertView
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    _adView = [[HyBidAdView alloc]initWithSize:HyBidAdSize.SIZE_320x50];
+    [self addSubview:self.adView];
+}
+
+- (nonnull instancetype)initWithSize:(VWAdSize)size {
+    
+    _adLoaded = false;
+
+    _adView = [[HyBidAdView alloc]initWithSize: [self mapSizes:size]];
+
+    [self addSubview:self.adView];
+
+    return self;
+}
+
+- (nonnull instancetype)initWithSize:(VWAdSize)size origin:(CGPoint)origin {
+    
+    _adLoaded = false;
+    
+    _adView = [[HyBidAdView alloc]initWithSize: [self mapSizes:size]];
+
+    CGRect frame = _adView.frame;
+    frame.origin = origin;
+    self.frame = frame;
+    
+    [self addSubview:self.adView];
+    
+    return self;
+}
+
+- (void)loadRequest:(nonnull VWAdRequest *)request {
+    [self.adView loadWithDelegate:self];
+}
+
+- (CGSize)sizeThatFits:(CGSize)size {
+    return [self sizeThatFits:size];
+}
+
+-(void) show {
```

Comment:
Social distancing between ) and "s" 👍 😂 But it would be better if it is between - and ( hahahah 😂

Reply:
usually I use swift lint for formatting code 💃